packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -10626,7 +10630,13 @@ export default function MyPostForm(props) {
           });
           await DataStore.save(
             Post.copyOf(postRecord, (updated) => {
-              Object.assign(updated, modelFields);
+              Object.assign(updated, {
+                ...modelFields,
+                nonModelFieldArray: modelFields.nonModelFieldArray.map((s) =>
+                  JSON.parse(s)
+                ),
+                nonModelField: JSON.parse(modelFields.nonModelField),
```

Comment:
if the nonModelField value is falsey, this will throw

Reply:
Thanks! will fix.